### DISCUSSION TITLE
Can I browse a recently opened files history?

### DISCUSSION THREAD

**Discussion by David-Else** [helix-editor/helix#1949] [Q&A]
I usually open Neovim and press 'space h' to open the `fzf.vim` file history so I can open the last file I was working on. I can't find anything like it in helix, is there some way of browsing recent files?

**Comment by kirawi** [comment]
This might be related to https://github.com/helix-editor/helix/issues/401? I'd like to expand the description of that issue, but I don't know enough about persistent state to be helpful :/

At any rate, I think this is a feature that can be implemented without relying on a complicated persistent state. What do you think, @archseer?

Edit: Actually, never mind. I think it might need to.
